src/app/login/login.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="login-page">
+    <section class="welcome-block">
+        <h2 class="welcome-title">Welcome</h2>
+        <p class="welcome-greetings">
+            To the Angular global mentoring program, here you can find courses
+            that cover all topics needed to start your own Angular application.
+        </p>
+    </section>
+    <section class="login-block">
+        <form class="login-form">
+            <h3 class="login-form-header">Login</h3>
+            <button class="external-login-btn">EPAM Login</button>
+            <div class="separator">or</div>
+            <div class="user-info">
+                <input class="user-email" type="email" placeholder="Email">
```

Comment:
there is no binding for enter values and component props, but it is asked in task